Use "oc get" without --export flag

michaelsauter opened the pull request:
Closes #229.

Adding quite some E2E tests, but some resources are still missing, and some E2E tests could do with more cases (e.g. hiding/revealing secrets, etc.). I will add them over time before the next release.

Manual testing also has shown no problems with using just `oc get` instead of `oc get --export`.

henrjk reviewed (approved):
I looked a bit into whether this may regress regarding issue #106, but as far as I can tell this should not be the case. 

The strategy to add more additional tests make sense. I also think that we should deliver this change soon and test it perhaps in existing projects.

michaelsauter commented:
Cool. I'm quite certain that the problem of #106 will not show up again due to the change here. When we switched from `oc export` to `oc get --export`, we already moved from the "export format" to the "get format", and to my understanding `--export` only differs to plain `get` infsofar  that some fields are removed that cannot be used to import the resource again.
